src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -1030,8 +1030,10 @@ export default function QuestionnaireEditor({ id }: QuestionnaireEditorProps) {
                   <div>
                     <CardTitle>
                       <p className="text-sm text-gray-700 font-medium mt-1">
-                        {questionnaire.questions?.length || 0} {t("question")}
-                        {questionnaire.questions?.length !== 1 ? t("s") : ""}
+                        {questionnaire.questions?.length || 0}{" "}
```

Comment:
Let's make use of the count parameter for translations (check entity_count key for example).